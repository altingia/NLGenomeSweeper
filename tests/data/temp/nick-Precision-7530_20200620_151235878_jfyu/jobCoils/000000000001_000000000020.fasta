>1
VTVSATTPLKDILRKSSKPSSIRLALQYPFMRILNVLILGSHPLLTIRSTIWLALSRWFS
LQNPSIKEVYVIVSGAMPRCIISLCSSSASRSFPSFTKLSIKTLNVTTLGLIFLLIISLN
NLAPSSHLPALQKPPMRML
>2
INRGSADNRDFHDNTKRHRFIRFFIIKRNEDNYKPGQISTNKFYPHIQTVPQFLKTPNKC
SFLIESSLPRSCRDLQKSQIQTMAEEAKAKGNAAFSSGDFTTAINHFTEAIALAPTNHVL
FSNRSAAHASLHQYAEALSDAKETIKLKPYWPKGYSRLGAAHLGLNQFELAVTAYKKGLD
VDPTNEALKSGLADAEASVARSRAAPNPFGDAFQGPEMWTKLTSDPSTRGFLQQPDFVNM
MQEIQKNPSSLNLYLKDQRVMQSLGVLLNVKFRPPPPQGDEAEVPESDMGQSSSNEPEVE
KKREPEPEPEPEVTEEKEKKERKEKAKKEKELGNAAYKKKDFETAIQHYSTAIEIDDEDI
SYLTNRAAVYLEMGKV
>3
LYFRIGRICSIPGNSAAYELKTSIALRHSCTILLHAMVSPMTLSANGRPQHFPATFLASS
GMSGWPFNVFSPTFIWNNSQASLGSRQLTSIGSSTRIRPQTSLERVVNTTLHPFCLLG
>4
KSYLSHLKINSSTICEVRDVFIINLNSSRIMLNSSFKVLLFIRSIPKLFLLFSFLLPLFL
LLLLCHLRFRFRLRFSLLLNLRLITRTLTHIRLRNLSLISLRRRWPKLNIQ
>5
KLQTRLSDPKFRCTEKKQKRETSDPESDLKTRKNRKMGKDGLSDDQVSSMKEAFMLFDTD
GDGKIAPSELGILMRSLGGNPTQAQLKSIIASENLSSPFDFNRFLDLMAKHLKTEPFDRQ
LRDAFKVLDKEGTGFVAVADLRHILTSIGEKLEPNEFDEWIKEVDVGSDGKIRYEDFIAR
MVAK
>6
SGNPLRLTFHDRSKTTSLGIDLKNSPEIFTRWRLSCRNKVVSLGHSGLPCISLILLFTSD
ILLTTFQSLTFGISFNLKPARTIQLSLCFLRSDAIHRAISRTTSCIFTYDLLLNFPPSTN
KEAQTRVPSIS
>7
CHCVLIQYNECIEDCNKAVERGRELRSDYKMVARALTRKGTALTKMAKCSKDYEPAIEAF
QKALTEHRNPDTLKRLNDAERAKKEWEQKQYFDPKLGDEEREKGDCFSN
>8
LIWAINQSNSQSTCLQETRLNKFRNLRTQSPYHQLHHLKLNRSFSNSHSHSVCKLQTLQN
SHGTRRTELRDKRNNTRNVPNKPVQFKLCRNWFFLYLTIKYGKIFIF
>9
TSHNRVTLVIRNSIQTKIASYSYNHNLRQSLFIYLSHYRICEDVLDFWVLHSLCLHKRKG
EIQVSFPGFLSITNGGRKHENHQNVTNLSLFQFLRSKVSAISLVDLLYTSAE
>10
SNITLRIAVNVISEENMIAICSEHNVSKHGKSAGHKSRIVADCKDIREKMNLPKRIFDVS
TLLSLFHLDDPSANKLLNLVHNHWILHVLLCCAWILLKIRQYLFGRKVLEC
>11
RFMVCSCNDFFLFFFVQCENLETPDAAELIAAMAAGWNVKLIVETWSEGDAIASSIGLNV
ASKHANAKHICIVQNSRSESAYLQAIQESSSPLNFPETIVAEEPGKAMKKLQGVDFLVVD
WRNKEFAAAALKNAAFGNRGAVVVCRNGYSSLRRVLRDRKVVRTVTLPVTGGIEIAHVAA
RNSGKSGNNKRRWITHVDQRSGEEHVFSI
>12
SHSIKISLRTKTLNLLQHQDNPFPIKIAHFHIFRAKSKETQPLKLSVISGKQFIEFILDG
RDIRKPSLDLRKPFTTPCELFYIYPSFHIISLFLQHFHVFLHSHYILGELANITHVDA
>13
EEDVIGWSKSYGFGEVVYSRGEVTGRESSVSFSFSFFCHCLNLRFLQVSAASWETTLYKK
RTLVWGFEKLRDCLDMRIEFVSGYLSGFIIILVTFNNKESYKTMSFCVVVEIPVISRPTI
YSPVPLFRVSFLRFDFVFVFTCSSVHLRNSESS
>14
LNFSSRLSFLVAFVNLSLCVKCKMGGCVSVQVSCDQLLNHLGRCFCRKLYYIQNIKENLT
SLEEAMEDLKALRDDLLRKVQTAEEGGLQRLHQIKVWLKRVKTIESQFNDLDSSRTVELQ
RLCCCGVGSRNLRLSYDYGRRVFLMLNIVEDLKSKGIFEEVAHPATRAVGEERPLQPTIV
GQETILEKAWDHLMDDGTKIMGLYGMGGVGKTTLLTQINNRFCDTDDGVEIVIWVVVSGD
LQIHKIQKEIGEKIGFIGVEWNQKSENQKAVDILNFLSKKRFVLLLDDIWKRVELTEIGI
PNPTSENGCKIAFTTRCQSVCASMGVHDPMEVRCLGADDAWDLFKKKVGDITLSSHPDIP
EIARKVAQACCGLPLALNVIGETMACKKTTQEWDRAVDVSTTYAANFGAVKERILPILKY
SYDNLESESVKTCFLYCSLFPEDDLIEKERLIDYWICEGFIDGDENKKGAVGEGYEILGT
LVCASLLVEGGKFNNKSYVKMHDVVREMALWIASDLRKHKDNCIVRAGFRLNEIPKVKDW
KVVSRMSLVNNRIKEIHGSPECPKLTTLFLQDNRHLVNISGEFFRSMPRLVVLDLSWNVN
LSGLPDQISELVSLRYLDLSYSSIGRLPVGLLKLKKLMHLNLESMLCLESVSGIDHLSNL
KTVRLLNLRMWLTISLLEELERLENLEVLTIEIISSSALEQLLCSHRLVRCLQKVSVKYL
DEESVRILTLPSIGDLREVFIGGCGMRDIIIERNTSLTSPCFPNLSKVLITGCNGLKDLT
WLLFAPNLTHLNVWNSRQIEEIISQEKASTADIVPFRKLEYLHLWDLPELKSIYWNPLPF
PCLNQINVQNKCRKLTKLPLDSQSCIVAGEELVIQYGDEEWKERVEWEDKATRLRFLPSC
KLVLCNR
>15
LRAPLNAKVADLYNSVGWTLAAPRTGLACDLLIYLSTIALPTDSDELDAYDWVVEDKLCN
GFSSAKTWEFSDQGRMWLAGQAQCGLKVQFPNMSLHMWATNLDRLPTKARLASWTLRTET
TCF
>16
ACFLDIEDRDHLFLTCEYACYIWYTVSVRLELPPINFVVWNDLMAWTLQKNKRSPPTTLR
KLIAQSVIYAIWKQRNNLHHNHEYILPSVIFKTIDREIKNSITARRLKKRFRRLMTLWLH
>17
IIVTHSIWFLVLLSRSLFSLFFPSPAMGACLTLSFSCDEVVNQISQGLCINVGYICELSK
NVVAMKKDMEVLKKKRDDVKRRVDIEEFTRRRERLSQVQGWLTNVSTVENKFNELLTTND
AELQRLCLFGFCSKNVKMSYLYGKRVVLMLKEIESLSSQGDFDTVTLATPIARIEEMPIQ
PTIVGQETMLERVWTRLTEDGDEIVGLYGMGGVGKTTLLTRINNKFSEKCSGFGVVIWVV
VSKSPDIHRIQGDIGKRLDLGGEEWDNVNENQRALDIYNVLGKQKFVLLLDDIWEKVNLE
VLGVPYPSRQNGCKVVFTTRSRDVCGRMRVDDPMEVSCLEPNEAWELFQMKVGENTLKGH
PDIPELARKVAGKCCGLPLALNVIGETMACKRMVQEWRNAIDVLSSYAAEFPGMEQILPI
LKYSYDNLNKEQVKPCFLYCSLFPEDYRMEKERLIDYWICEGFIDENESRERALSQGYEI
IGILVRACLLLEEAINKEQVKMHDVVREMALWIASDLGEHKERCIVQVGVGLREVPKVKN
WSSVRRMSLMENEIEILSGSPECLELTTLFLQKNDSLLHISDEFFRCIPMLVVLDLSGNS
SLRKLPNQISKLVSLRYLDLSWTYIKRLPVGLQELKKLRYLRLDYMKRLKSISGISNISS
LRKLQLLQSKMSLDMSLVEELQLLEHLEVLNISIKSSLVVEKLLNAPRLVKCLQILVLRG
VQEESSGVLTLPDMDNLNKVIIRKCGMCEIKIERKTLSLSSNRSPKTQFLHNLSTVHISS
CDGLKDLTWLLFAPNLTSLEVLDSELVEGIINQEKAMTMSGIIPFQKLESLRLHNLAMLR
SIYWQPLSFPCLKTIHITKCPELRKLPLDSEIAIRDEELVIKYQEEEWLERVEWDNEATR
LRFLPFFKFFGPEWQVSYVR
>18
ESISKKLFFNLPSDNISITIFIVEASTENPHLFSSSISLTDLVASPSPRCALMSMLYVQP
SGACPSSSIFLNNSSASDNGPFLHSPPIIILYVLTPGFTPLRGRLQNKSQASSTLLALHI
P
>19
KIRSHLATILAMKSSYRIFPSDPTSTSLIHSSNSLGSSFSPILVRICLRSATATNPVPSL
SSTLNASRSWRSKGSVFRCFAMRSRNLLKSNGEERFSDAIMDFSWAWVGFPPRDRMRIPS
SDGAILPSPSVSKSMKASFIDET
>20
KGTIQLEMVKLMIRRLSSQVSKFVQPRLLETGTLRIALINCPNELSFCCERGFSAFSDRN
LSYRERLRSGLVDIKADDAIDLFRDMIHSRPLPTVIDFSRLFSAIAKTKQYDLVLALCKQ
MELKGIAHNLYTLSIMINCFCRCRKLCLAFSAMGKIIKLGYEPNTITFSTLINGLCLEGR
VSEALELVDRMVEMGHKPDLITINTLVNGLCLSGKEAEAMLLIDKMVEYGCQPNAVTYGP
VLNVMCKSGQTALAMELLRKMEERNIKLDAVKYSIIIDGLCKHGSLDNAFNLFNEMEMKG
ITTNIITYNILIGGFCNAGRWDDGAKLLRDMIKRKINPNVVTFSVLIDSFVKEGKLREAE
ELHKEMIHRGIAPDTITYTSLIDGFCKENHLDKANQMVDLMVSKGCDPNIRTFNILINGY
CKANRIDDGLELFRKMSLRGVVADTVTYNTLIQGFCELGKLNVAKELFQEMVSRKVPPNI
VTYKILLDGLCDNGESEKALEIFEKIEKSKMELDIGIYNIIIHGMCNASKVDDAWDLFCS
LPLKGVKPGVKTYNIMIGGLCKKGPLSEAELLFRKMEEDGHAPDGWTYNILIRAHLGDGD
ATKSVKLIEELKRCGFSVDASTIKMVIDMLSDGRLKKSFLDMLS
