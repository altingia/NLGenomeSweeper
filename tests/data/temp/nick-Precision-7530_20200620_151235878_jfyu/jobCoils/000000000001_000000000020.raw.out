>1
//
>2
//
>3
//
>4
//
>5
//
>6
//
>7
69 89
//
>8
//
>9
//
>10
//
>11
//
>12
//
>13
//
>14
52 86
//
>15
//
>16
//
>17
62 82
//
>18
//
>19
//
>20
//
